Project.toml

```diff
@@ -18,34 +18,38 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MarchingCubes = "299715c1-40a9-479a-aaf9-4a633d36f717"
 NaNMath = "77ba4419-2d1f-58cd-9bb1-8ffee604a2e3"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
-ColorSchemes = "3.19"
+ColorSchemes = "≥ 3.19"
```

Comment:
It's not a good idea to allow arbitrary breaking releases with `>=`. You can't ensure this package works with ColorSchemes v4, or Crayons v5 unless they're released.

A lot of Julia packages used this, and then the entire ecosystem shifts to the more restricted version. https://github.com/JuliaRegistries/General/pull/11114

Reply:
Do you say that all the dependencies like `StaticArrays`, `Unitful`, ... currently at `1` should be pinned to a minor range instead ?

Reply:
> https://pkgdocs.julialang.org/v1/compatibility/#Version-specifier-format
> Caret specifiers are the default and hence 1.2.3 == ^1.2.3. 

I was speaking specifically about the `>=` usage (can't mind reading and realize that `>=` is a typo). Caret is okay, but `^3.19` equivalent to `3.19`.

Reply:
I will revert to caret, sorry.  `^3.19` means `[3.19.0, 4.0.0)` according to https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers, which is what I originally meant.